wiki/en/en-Installation-for-Macintosh.md

```diff
@@ -10,14 +10,15 @@ permalink: "/wiki/Installation-for-Macintosh"
 Make sure you've already read the [Getting Started](Getting-Started) page.
 
 1. [Download Jamulus](https://sourceforge.net/projects/llcon/files/latest/download){: target="_blank" rel="noopener noreferrer" .button}
-1. **Open the downloaded `.dmg` file**. Agree to the license. You will now see two icons (Jamulus client and server).
-1. **Drag and drop both icons into your Applications folder** to install Jamulus.
+1. **Install Jamulus**. Open the downloaded `.dmg` file, agree to the license, *drag and drop* the two icons you see in the window (Jamulus client and server) into your *Applications folder*. After that, you can close this window.
 1. **Run Jamulus**. Now you should be able to use Jamulus just like any other application. If you see a message which tells you that Jamulus can't be opened because the developer can't be verified, read on and have a look at the following section.
 
 _You can remove the folder in the Downloads directory containing the `.dmg` and eject the "Jamulus" drive on your desktop. They are no longer needed._
 
 You can also [compile your own binary](Compiling#macos) if you feel so inclined.
```

Comment:
Since less than about 1% of anyone reading this page will know how to do this, and those that do will know that Jamulus is FOSS, I'm inclined to remove this line. It might even confuse some people.

Reply:
@gilgongo I only see the problem for more experienced users: they would have to search for it on https://jamulus.io/wiki/ we might have to add it to the navigation

Reply:
We can put it in a footnote if you really want, but those wanting to compile Jamulus on Windows will just go to Github I think as they are likely to be contributors to the project anyway. I highly doubt anyone would go to the trouble of compiling Jamulus on Windows for any other reason.

Reply:
Do we want all users to install both the client and server software (from line 14: "* Drag and drop both icons"). It seems to me most users only need to install the client. I don't want to be helping naive users with server questions. The server issues are hard enough for me.

Reply:
Heh - well, on Windows they get it whether they want it or not. You raise a good point, but it's hard to know what to say here without diverging into some kind of discussion about which to install and why (and I think we don't label the icon "Jamulus client" which makes referring to it harder).

Reply:
> I highly doubt anyone would go to the trouble of compiling Jamulus on those platforms for any other reason.

Ok. Valid point. We might need to add the overview page link somewhere more prominent. 

@gene96817 

> It seems to me most users only need to install the client.

Probably true. Also the server on macOS seems to be a own binary? I think windows and Linux only have one binary? So the user would even benefit from less space being used on his hard drive.